Xabe.FFmpeg/Probe/Probe.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Threading.Tasks;
+
+namespace Xabe.FFmpeg
+{
+    /// <inheritdoc />
+    public class Probe : IProbe
+    {
+        /// <summary>
+        ///     Get new instance of Conversion
+        /// </summary>
+        /// <returns>IProbe object</returns>
+        public static IProbe New()
+        {
```

Comment:
Why it cannot  be instantiated by constructor?

Reply:
To be compatible with Conversion.New() ;) Constructor is not private here so can be instantiated by constructor.